src/Symfony/Component/Messenger/composer.json

```diff
@@ -20,7 +20,7 @@
         "psr/log": "~1.0"
     },
     "require-dev": {
-        "doctrine/dbal": "^2.5",
+        "doctrine/dbal": "^2.6",
```

Comment:
should we conflict with <2.6 too or is it fine to still use e.g. 2.5?

Reply:
It works with 2.5 as well. It just wouldn't have the RetryableException (https://github.com/doctrine/dbal/pull/718) so behave like before this PR.